src/Sarif.Multitool.Library/MergeCommand.cs

```diff
@@ -307,6 +318,11 @@
 
             return prefix ?? string.Empty;
         }
+
+        private static string ReplaceInvalidCharInFileName(string fileName, string replacement)
```

Comment:
>ReplaceInvalidCharInFileName [](http://example.com/codeflow?start=30&length=28)

You provided no unit tests for this helper. Generally, a helper with a single line of code with a single upstream caller doesn't warrant dedicating its own method.

Reply:
dedicated unit tests added pls review